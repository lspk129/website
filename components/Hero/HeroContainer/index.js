import styled from 'styled-components';

const HeroContainer = styled.div`
  padding-bottom: 100px;
  grid-row: 2 / 3;
  justify-self: center;
  text-align: center;
`;

export default HeroContainer;
