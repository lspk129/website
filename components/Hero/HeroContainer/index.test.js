import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import HeroContainer from './index';

describe('HeroContainer', () => {
  it('renders corretly', () => {
    const wrapper = renderer.create(<HeroContainer />);
    const tree = wrapper.toJSON(wrapper);
    expect(tree).toMatchSnapshot();
  });

  it('renders div element', () => {
    const wrapper = shallow(<HeroContainer />);
    expect(wrapper).toBePresent();
    expect(wrapper).toHaveTagName('div');
  });
});
