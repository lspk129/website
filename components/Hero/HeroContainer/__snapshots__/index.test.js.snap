// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroContainer renders corretly 1`] = `
<div
  className="sc-bdVaJa hLmcko"
/>
`;
